### ISSUE TITLE
onPhoneNumberChange/onPhoneNumberBlur has country code twice when separateDialCode={true}

### ISSUE THREAD

**Issue by nyanpasu** [patw0929/react-intl-tel-input#208]
## Sample test

The onPhoneNumberChange/onPhoneNumberBlur event handlers are passing incorrect values when separateDialCode is true.

```
import React from 'react';
import IntlTelInput from 'react-intl-tel-input';
import 'react-intl-tel-input/dist/libphonenumber';
import 'react-intl-tel-input/dist/main.css';

class App extends React.Component {
  render() {
    return (
      <div>
        <IntlTelInput 
          preferredCountries={['tw']}
          css={['intl-tel-input', 'form-control']}
          utilsScript={'libphonenumber.js'} 

          onSelectFlag={(num, country) => {
            console.log('onSelectFlag', num, country);
          }}
          onPhoneNumberChange={(status, value, countryData, number, id) => {
            console.log('onPhoneNumberChange value', value);
            console.log('onPhoneNumberChange number', number);
          }}
          onPhoneNumberBlur={(status, value, countryData, number, id) => {
            console.log('onPhoneNumberBlur value', value);
            console.log('onPhoneNumberBlur number', number);
          }}
          format
          formatOnInit={false}
          separateDialCode
        />
      </div>
    );
  }
}

export default App;

```

## Sample output

Without doing anything, and just loading the page, this gets logged:

```
onPhoneNumberChange value +886
onPhoneNumberChange number +886+886
```

I would expect `value` to be to be `null` or `''` (so the console should not log anything, instead of `+886`) and `number` to be `+886`, not `+886+886`

As I type some more numbers, I see:

```
onPhoneNumberChange value +8861
onPhoneNumberChange number +886+8861
onPhoneNumberChange value +886 12
onPhoneNumberChange number +886+886 12
onPhoneNumberChange value +886 123
onPhoneNumberChange number +886+886 123
onPhoneNumberBlur value +886 123
onPhoneNumberBlur number +886+886 123
```

## Info

```
    "react": "^16.3.2",
    "react-dom": "^16.3.2",
    "react-intl-tel-input": "^5.0.4"
```

**Comment by nyanpasu** [comment]
Demo: https://yml35835j1.codesandbox.io/

**Comment by patw0929** [comment]
Made fix commits #209 & #210.
Could you help to check? If everything is ok, I'll release new version.

**Comment by nyanpasu** [comment]
lgtm on the master branch. Thanks for the fix!

**Comment by patw0929** [comment]
No problem! Released as v5.0.5.
